src/components/pages/StudyRights.tsx

```diff
@@ -15,11 +18,16 @@ interface Props {
   user: User
 }
 
-const StudyRightsPage = async ({ study, user }: Props) => {
-  const tNav = await getTranslations('nav')
-  const t = await getTranslations('study.rights')
+const StudyRightsPage = ({ study, user }: Props) => {
+  const tNav = useTranslations('nav')
+  const t = useTranslations('study.rights')
 
-  const userRoleOnStudy = getUserRoleOnStudy(user, study)
+  const userRoleOnStudy = getUserRoleOnStudy(user, study) // attendre la fusion de la PR de Louis
+
+  const editionDisabled = useMemo(
+    () => !isAdminOnStudyOrga(user, study) && userRoleOnStudy === StudyRole.Reader,
```

Comment:
Il y a déjà un test isAdminOnStudyOrga dans le userRoleOnStudy
```suggestion
    () => !userRoleOnStudy ||  userRoleOnStudy === StudyRole.Reader,
```